rotkehlchen/api/v1/schemas.py

```diff
@@ -620,7 +620,9 @@ class HistoryBaseEntrySchema(Schema):
         load_default=HistoryEventSubType.NONE,
         allow_none=True,
     )
-    counterparty = fields.String(required=False)
+    counterparty = fields.String(load_default=None)
```

Comment:
Yes, I made it `protocol`, but later, since not only protocols, but also exchange names like `kraken` can be here Yabir said to rename it to counterprty.